index.html

```diff
@@ -61,12 +61,14 @@
             <td><input type="button" value="1" onclick="dis('1')"/> </td> 
             <td><input type="button" value="2" onclick="dis('2')"/> </td> 
             <td><input type="button" value="3" onclick="dis('3')"/> </td> 
+            <td><input type="button" value="/" onclick="dis('/')"/> </td> 
```

Comment:
this should have been done in a drop down box with a switch statement attached to it. These are all tools you learned in class.  This is coming from a tutorial that you were not taught and most likely couldn't replicate.